frontend/components/Sandbox/SandboxExpiryMessage/SandboxExpiryMessage.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+import ExternalURLIcon from "../../../../assets/images/icon-external-url-black-12x12@2x.png";
+
+const baseClass = "sandbox-expiry-message";
+
+interface ISandboxExpiryMessageProps {
+  expiry: string;
+}
+
+const SandboxExpiryMessage = ({
+  expiry,
+}: ISandboxExpiryMessageProps): JSX.Element => {
+  return (
+    <div className={baseClass}>
+      <p>Your Fleet Sandbox expires in {expiry}.</p>
+      <a
+        href="https://fleetdm.com/docs/deploying"
```

Comment:
Don't forget the Fleet Premium expiriation is moving from the flash message to this banner component as well

Reply:
<img width="527" alt="Screen Shot 2022-07-25 at 3 40 02 PM" src="https://user-images.githubusercontent.com/71795832/180860884-84e787db-34d5-46c9-8193-eda0663a195b.png">

Reply:
this was something i chatted to noah about last week and since this was going to be a larger change, that was the last bit i was gonna try to do when the feature was done if there was time. Looks like there will be enough time so I will change this in another PR.